src/DevBetterWeb.Core/Interfaces/IPaymentHandlerSubscriptionDTO.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace DevBetterWeb.Core.Interfaces
+{
+  public interface IPaymentHandlerSubscriptionDTO
```

Comment:
DTOs don't typically need interfaces. Is there a reason for this one vs just a class?